components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/api/APIGateway.java

```diff
@@ -155,4 +156,24 @@ void updateAPISubscriptionStatus(List<SubscriptionValidationData> subscriptionVa
      */
     void deleteApplication(String applicationId) throws GatewayException;
 
+    /**
+     * Publish policy add event to Gateway
+     * @param policyValidationData policy Data
```

Comment:
Javadoc formatting is not correct. There should be a extra new line between description and params